src/sidebar/components/MentionSuggestionsPopover.tsx

```diff
@@ -21,6 +71,12 @@ export type MentionSuggestionsPopoverProps = Pick<
   onSelectUser: (selectedSuggestion: UserItem) => void;
   /** Element ID for the user suggestions listbox */
   usersListboxId: string;
+
+  /**
+   * Default service authority, used to determine if mention suggestions are
+   * third party users or not
+   */
+  defaultAuthority: string;
```

Comment:
At this level, I think the configuration we want to pass is just whether the mentions popover should show usernames or not. The current implementation allows for deciding on a per-annotation basis whether to show the username, but we can simplify this if we assume that we show usernames either for all users in the suggestions or none.